### ISSUE TITLE
Nimbus stops respond in windows 10 after a while

### ISSUE THREAD

**Issue by rubensx** [Axadiw/MFIGamepadFeeder#56]
using nimbus steelseries after a some minutes the controller loose respond, in game without a warning or a disconnect sound 

someone know what happens?

**Comment by julianpetriv** [comment]
Same problem here. What version of Windows 10 you have?

**Comment by rubensx** [comment]
2004 last version may 2020

**Comment by julianpetriv** [comment]
I think that this problem is connected with this windows version. And as of May 10, when you opened this issue you were running Windows Insider? Cause 2004 was only released on May 27.

**Comment by rubensx** [comment]
No, the problem comes from older versuons and still are same 1904 and 1800
